src/figuresClasses.ts

```diff
@@ -1,11 +1,80 @@
-export interface Figure {}
+export interface Figure {
+  getArea: () => number;
+  shape: string;
+  color: string;
+}
+
+export class Triangle implements Figure {
+  public shape: string;
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    this.shape = 'triangle';
+
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('your error message');
```

Comment:
[CODE STYLE] - When throwing an error, provide a detailed description in the error message to make it clear what went wrong. For example, instead of 'your error message', you could use 'Triangle side lengths must be greater than zero.'